pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -42,18 +42,19 @@
  */
 public class GrpcSendingMailbox implements SendingMailbox<TransferableBlock> {
   private static final Logger LOGGER = LoggerFactory.getLogger(GrpcSendingMailbox.class);
-  private final String _mailboxId;
-  private final AtomicBoolean _initialized = new AtomicBoolean(false);
 
-  private StreamObserver<MailboxContent> _mailboxContentStreamObserver;
+  private final String _mailboxId;
```

Comment:
shouldn't this be MailboxIdentifier type?

Reply:
I kept it as string here because we need to include it as string in the mailbox content. This can avoid serializing it multiple times